Align docs to changes in the testing environment (#1912)
ckeditor/ckeditor5: docs/framework/guides/contributing/testing-environment.md

* `--verbose` (alias `-v`) &ndash; Allows switching on webpack logs.
* `--files` &ndash; Specifies tests files to run. Accepts a package name or a glob. Read more about the [rules for converting the `--files` option to a glob pattern](https://github.com/ckeditor/ckeditor5-dev/tree/master/packages/ckeditor5-dev-tests#rules-for-converting---files-option-to-glob-pattern).
* `--browsers` &ndash; Browsers which will be used to run the tests. Defaults to `Chrome`.

> * `--debug` (alias `-d`) &ndash; Allows specifying custom debug flags. For example, the `--debug engine` option uncomments the `// @if CK_DEBUG_ENGINE //` lines in the code. The `--debug false` turns off the debugging mechanism and leaves the lines starting with `// @if CK_DEBUG //` as they are.

Reviewer: ```suggestion
* `--debug` (alias `-d`) &ndash; Allows specifying custom debug flags. For example, the `--debug engine` option uncomments the `// @if CK_DEBUG_ENGINE //` lines in the code. The `--debug false` option turns the debugging mechanism off and leaves the lines starting with `// @if CK_DEBUG //` untouched.
```

> * `--debug` (alias `-d`) &ndash; Allows specifying custom debug flags. For example the `--debug engine` option uncomments the `// CK_DEBUG_ENGINE //` lines in the code. The `--debug false` turns off debugging (by default uncomments lines starting with `// CK_DEBUG //` in manual and automated tests).

Author: ```suggestion
* `--debug` (alias `-d`) &ndash; Allows specifying custom debug flags. For example, the `--debug engine` option uncomments the `// @if CK_DEBUG_ENGINE //` lines in the code. The `--debug false` turns off the debugging mechanism and leaves the lines starting with `// @if CK_DEBUG //` as they are.
```
